plugin/geoip/README.md

```diff
@@ -16,35 +18,54 @@ import (
 // ...
 if getLongitude := metadata.ValueFunc(ctx, "geoip/longitude"); getLongitude != nil {
     if longitude, err := strconv.ParseFloat(getLongitude(), 64); err == nil {
-		// Do something useful with longitude.
-	}
+        // Do something useful with longitude.
+    }
 } else {
     // The metadata label geoip/longitude for some reason, was not set.
 }
 // ...
 ```
 
 ## Databases
+
 The supported databases use city schema such as `City` and `Enterprise`. Other databases types with different schemas are not supported yet.
 
 You can download a [free and public City database](https://dev.maxmind.com/geoip/geolite2-free-geolocation-data).
 
 ## Syntax
-```txt
+
+```text
 geoip [DBFILE]
 ```
+
+or
+
+```text
+geoip [DBFILE] {
+    edns-subnet-processing
+}
+```
+
+By default, geoip plugin doesn't honor EDNS0 subnet extension, but it can be enabled using the extended syntax:
```

Comment:
```suggestion
```

Reply:
huh did that line make it so far?

Reply:
I think that this is described sufficiently in the syntax section, and doesn't require more documentation.